server/ec2spotmanager/common/ec2.py

```diff
@@ -0,0 +1,17 @@
+REGIONS = (
+    "ap-northeast-1",
+    "ap-northeast-2",
+    "ap-south-1",
+    "ap-southeast-1",
+    "ap-southeast-2",
+    "ca-central-1",
+    "eu-central-1",
+    "eu-west-1",
+    "eu-west-2",
+    "eu-west-3",
+    "sa-east-1",
+    "us-east-1",
+    "us-east-2",
+    "us-west-1",
+    "us-west-2"
+)
```

Comment:
Would it make sense to pull this information from our pool configurations instead? Otherwise, we will soon run into the problem that we have to update this file whenever a region is added or removed. Querying the database for all regions in use should be fairly quick.